util/dockerutil/docker.go

```diff
@@ -34,7 +34,9 @@ func NewDockerClient() (*client.Client, error) {
 			}
 			// Error message example:
 			//   Error getting metadata for container: Error response from daemon: client is newer than server (client API version: 1.26, server API version: 1.24)
-			os.Setenv("DOCKER_API_VERSION", version[1])
+			// Hardcoding Docker version until version[1] returns valid version
+			// os.Setenv("DOCKER_API_VERSION", version[1])
+			os.Setenv("DOCKER_API_VERSION", "1.24")
```

Comment:
This seems like something that will come back to bite you. 

You could try initializing the docker client the new way: https://pkg.go.dev/github.com/docker/docker/client#NewClientWithOpts